server/exercise/src/main/scala/com/eigengo/lift/exercise/classifiers/workflows/ClassificationAssertions.scala

```diff
@@ -9,7 +9,7 @@ object ClassificationAssertions {
    * Named gesture matches with probability >= `matchProbability`
    */
   def Gesture(name: String, matchProbability: Double, sensor: SensorDataSourceLocation): GroundFact =
-    new GroundFact(name, Some((matchProbability, sensor))) {
+    new GroundFact(name, matchProbability, sensor) {
```

Comment:
I'm not sure what you're getting at here?

To see why I might argue that it does work, have a look at `ExerciseModelTest`. In those tests we setup a number of similar case class instances and use them in testing. Moreover, equality testing is performed on them.